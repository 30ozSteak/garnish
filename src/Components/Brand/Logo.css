.logo-box {
  margin: 1rem;
}

.logo-box .ish {
  margin-left: 2px;
  transition: 0.4s;
}

.logo-box .ish:hover {
  box-shadow: 2px 2px 10px lightgray;
  cursor: pointer;
  filter: brightness(105%);
}
