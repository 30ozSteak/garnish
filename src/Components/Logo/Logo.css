.logo-box {
  margin: 1rem;
}

.logo-box .ish {
  margin-left: 2px;
  transition: 0.4s;
}

.logo-box .ish:hover {
  box-shadow: 2px 2px 10px lightgray;
  cursor: pointer;
  filter: brightness(105%);
}

.bigger-slot-box a {
  color: darkgray;
  position: absolute;
  text-decoration: none;
  transition: 0.2s;
  top: 40rem;
}

.bigger-slot-box a:hover {
  color: black;
}

.bigger-slot-box {
  text-align: center;
}
