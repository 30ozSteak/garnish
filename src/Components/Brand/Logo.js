import React from "react";
import "../Brand/Logo.css";

const Logo = () => {
  return (
    <div className="logo-box">
      <h3 className="app-title">
        garn
        <span className="ish">ish</span>
      </h3>
    </div>
  );
};

export default Logo;
