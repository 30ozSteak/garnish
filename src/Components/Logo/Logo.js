import React from "react";

import "../Logo/Logo.css";

const Logo = () => {
  return (
    <div className="logo-box">
      <h3 className="app-title">
        <span className="ish">ish</span>
      </h3>
    </div>
  );
};

export default Logo;
