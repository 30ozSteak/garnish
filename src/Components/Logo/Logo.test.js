import React from "react";
import Logo from "./Logo";
import { shallow } from "enzyme";

describe("Logo", () => {
  it("should resemble the snapshot", () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });
});
