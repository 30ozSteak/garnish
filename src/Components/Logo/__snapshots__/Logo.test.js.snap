// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should resemble the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Logo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h3
        className="app-title"
      >
        <span
          className="ish"
        >
          ish
        </span>
      </h3>,
      "className": "logo-box",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <span
          className="ish"
        >
          ish
        </span>,
        "className": "app-title",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "ish",
          "className": "ish",
        },
        "ref": null,
        "rendered": "ish",
        "type": "span",
      },
      "type": "h3",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h3
          className="app-title"
        >
          <span
            className="ish"
          >
            ish
          </span>
        </h3>,
        "className": "logo-box",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <span
            className="ish"
          >
            ish
          </span>,
          "className": "app-title",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "ish",
            "className": "ish",
          },
          "ref": null,
          "rendered": "ish",
          "type": "span",
        },
        "type": "h3",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
